tests/test_simulation_metadata.py

```diff
@@ -92,10 +99,12 @@ def test_get_metadata_all(self):
         metadata_api = self.sim.get_metadata()
 
         # Access the metadata directly in the frame.
-        metadata_df = self.sim._metadata.to_numpy()
+        metadata_array = np.vstack(
```

Comment:
I think because the table is a table of objects (arrays).

This test should just be checking that when we ask for the entire table we get the same thing as concatenating asking for each column (metadata key) .  A list could have been used instead of a vstack.  Not sure if that answers your concern.

Reply:
Ok got it.